DESCRIPTION

```diff
@@ -53,4 +53,4 @@ Suggests:
     testthat (>= 3.0.0)
 Config/testthat/edition: 3
 Depends: 
-    R (>= 2.10)
+    R (>= 3.5.0)
```

Comment:
FYI that I [removed this to-do from the roadmap](https://github.com/JMSLab/eventstudyr/wiki/Roadmap/_compare/f38e9ef7bb955eb57a5182dff03e9d97c465b3ce...decc4f7c70eecd1c8427adf8bfbe9966f62c35f5)